src/router_core/connections.c

```diff
@@ -1456,42 +1456,48 @@ void qdr_process_addr_attributes_CT(qdr_core_t *core, qdr_address_t *addr)
 static void qdr_connection_group_setup_CT(qdr_core_t *core, qdr_connection_t *conn)
 {
     const char *correlator = conn->connection_info->group_correlator;
+    uint64_t    ordinal    = conn->connection_info->group_ordinal;
 
     assert(conn->role == QDR_ROLE_INTER_ROUTER);
     qd_log(LOG_ROUTER_CORE, QD_LOG_DEBUG,
-           "CGROUP qdr_connection_group_setup_CT - %lx (%s)",
-           (ulong) conn, conn->connection_info->host);
+           "CGROUP setup group '%s' for parent [C%"PRIu64"] with ordinal=%"PRIu64" (host=%s)",
+           correlator, conn->identity, ordinal, conn->connection_info->host);
 
     if (strnlen(correlator, QD_DISCRIMINATOR_SIZE) > 0) {
+#if 0   // KAG TODO FIX: allow dupes for now:
```

Comment:
Should this `TODO` be resolved ?

Reply:
Actually that TODO gets fixed and removed in the next patch #1748 
For now we need to allow dups because the control links are not removed yet.